packages/api/src/context.ts

```diff
@@ -667,7 +667,7 @@ export async function contextFromRequest(
               )?.value as number) ||
               parseInt(process.env.PEERING_TIMEOUT_IN_MS as string) ||
               3000
-            const fetcher = createFetcher(peer.hostURL, peer.token, peerTimeout)
+            const fetcher = createFetcher(peer?.hostURL ?? '', peer?.token ?? '', peerTimeout)
```

Comment:
Don't think this change is needed (the peer?.) as if peer is not defined we return

Should we maybe also check if hostURL or token is set? Else we might make an HTTP call we didn't want to.